app/views/dataset.scala.html

```diff
@@ -382,6 +382,24 @@ <h4>Statistics</h4>
     <script src="@routes.Assets.at("javascripts/deleteUtils.js")" type="text/javascript"></script>
     <script src="@routes.Assets.at("javascripts/people.js")" type="text/javascript"></script>
     <script src="@routes.Assets.at("javascripts/creators.js")" type="text/javascript"></script>
+
+    <script type="application/ld+json">{"context": { "@@vocab": "https://schema.org/"},
```

Comment:
```suggestion
    <script type="application/ld+json">
      @dataset.to_jsonld().stringyfy()
    </script>
```
then have in each class a json_ld function that will make it jsonld, all that is needed at that last point is to add a \@context

Reply:
yes moved that whole block in dataset.scala.html to to a Dataset.scala method, leaving only script tags; will test a bit 1st though

Reply:
classes and a few it links to all have to_jsonld methods